franka_gazebo/package.xml

```diff
@@ -43,5 +43,9 @@
   <export>
     <controller_interface plugin="${prefix}/franka_gripper_sim_plugin.xml"/>
     <gazebo_ros_control   plugin="${prefix}/franka_hw_sim_plugin.xml"/>
+
+    <!--Export gazebo models and worlds-->
+    <gazebo_ros gazebo_model_path="${prefix}/models"/>
+    <gazebo_ros gazebo_resource_path="${prefix}/worlds"/>
```

Comment:
path needs to be changed to "world" when the "worlds" folder is renamed to "world".
Out of curiosity: What is this actually doing. Is this somewhere documented?

Reply:
It is a trick to set several GAZEBO env for the ROS package (see http://gazebosim.org/tutorials?tut=components). By doing this Gazebo also looked for worlds and models inside our package. As a result, we can now use relative paths instead of substitution args (i.e. `$(find franka_gazebo)/world/stone.world`). It is documented [here](http://gazebosim.org/tutorials?tut=ros_roslaunch). See the `Exporting model paths from a package.xml` section. It would allow you to update the roslaunch argument in the documentation:

``` diff
roslaunch franka_gazebo panda.launch x:=-0.5 \
-  world:=$(rospack find franka_gazebo)/world/stone.sdf \
+ world:=world.stone.sdf \
   controller:=cartesian_impedance_example_controller \
   rviz:=true
```

It is just a matter of preference. I can change it if you prefer the more explicit syntax.

Reply:
@marcbone Apparently this environmental variable only works inside other launch/world or model files. The above mentioned roslaunch command still needs the $(rospack find franka_gazebo) prefix. It is needed to specify models inside the `stone.sdf` world file:

https://github.com/frankaemika/franka_ros/blob/b5d4dc6cd87af3e4b4af3817904fcc8b7d6316f1/franka_gazebo/world/stone.sdf#L14